cypress/integration/login.js

```diff
@@ -1,28 +1,114 @@
+import LoginPage from '../objects/logingPage'
+import TimeLogingPage from '../objects/timeLogingPage'
+const loginPage = new LoginPage()
+const timeLogingPage = new TimeLogingPage()
 describe('Sourcebooks login', function() {
 
     it('Should display validation for empty user after attempted loggin', function () {
         
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
+
+        loginPage.visit();
+        
+        loginPage.getValidationError().should('not.visible')
+        loginPage.getSubmit().click()
+        loginPage.getValidationError().should('be.visible')
+    })
+
+    it('Should be able to login with role User', function () {
+
+        loginPage.visit();
+        loginPage.getUserForm().click({force:true})
+        loginPage.getUserOption("Jevgenija Malomuž").click();
+        loginPage.getLoginForm().click({force:true});
+        loginPage.getLabel('User').click();
+        loginPage.getSubmit().click()
+
+        timeLogingPage.getUrl('/time-logging');
+        timeLogingPage.getPageTitle().contains('Timesheets')
+        timeLogingPage.getCalendarVisible().should('be.visible')
+        timeLogingPage.getCalendarToday().contains(new Date().getDate());
+        timeLogingPage.getTileForm().should('be.visible')
+        timeLogingPage.getInfoTitle().contains('Jevgenija Malomuž');
+        timeLogingPage.getLength().find('li').should('have.length', 1);
+        timeLogingPage.getTimeLoggingColor().contains('Time Logging').should('have.css' , 'color' , 'rgb(64, 76, 237)');
+
+    })
+
+    it('Should be able to login with role User', function () {

```

Comment:
This test case should say 'role Team Lead' and not 'User', right? And then the ones below should use the corresponding roles in their names too